### ISSUE TITLE
Error when autoformatting html file

### ISSUE THREAD

**Issue by JacobWay** [Chiel92/vim-autoformat#139]
Installed js-beautify plugin globally,
Set the path of formatter folder in .vimrc file: 
  let g:formatterpath = ['/usr/local/bin']

But when I running the command :Autoformat, there are errors below:

Error detected while processing function <SNR>40_TryAllFormatters:
line   35:
E117: Unknown function: shiftwidth
Formatter htmlbeautify has errors: /bin/sh: 0: command not found
 Skipping.
Failing config: '0'
Definition in 'g:formatdef_htmlbeautify' was unsuccessful.
E117: Unknown function: shiftwidth
Formatter tidy_html has errors: /bin/sh: 0: command not found
 Skipping.
Failing config: '0'
Definition in 'g:formatdef_tidy_html' was unsuccessful.
No format definitions were successful.
Removing trailing whitespace...
Retabbing...
Autoindenting...

How to solve the error? Thanks for your reply.

**Comment by Chiel92** [comment]
What is your vim version?

**Comment by JacobWay** [comment]
VIM - Vi IMproved 7.3 (2010 Aug 15, compiled Nov 12 2015 15:12:18) @Chiel92

**Comment by Chiel92** [comment]
I believe the `shiftwidth()` function is only available since 7.4.  I recommend upgrading.

**Comment by Chiel92** [comment]
If that's not possible for whatever reason, you could also write a patch that uses the function

```vim
if exists('*shiftwidth')
  func s:sw()
    return shiftwidth()
  endfunc
else
  func s:sw()
    return &sw
  endfunc
endif
```

instead of `shiftwidth()`. I would accept such a pull request.
